CONTRIBUTING.md

```diff
@@ -191,3 +191,41 @@ these steps:
 >    ```
 >    The error is related to the `tcl` package. You should uninstall the `tcl`
 > package to resolve the error.
+
+## Error handler
+
+Use the `DPCTL_VERBOSITY` environment variable to show errors of different severity levels of the C-API in the console.
+You can specify one of the following severity levels (in increasing order of severity): `warning` and `error`.
+
+```bash
+export DPCTL_VERBOSITY=error
+```
+
+Messages of a given severity are shown not only in the console for that severity, but also for a lower severity.For example, the severity level `error` will output severity errors for `error` and `warning` to the console.
+
+## Google logging libriry (glog)
+
+The dpctl error handler provides logging for exceptions of different severity levels in `libsyclinterface` using the [glog](https://github.com/google/glog) library.
+To use logging during development , follow these steps:
+
+1. Install glog package of the latest version (0.5.0)
+     ```bash
+    conda install glog
+    ```
+2. Build dpctl with glog support
+     ```bash
+    python scripts/build_locally.py --oneapi --glog
+    ```
+3. Use `DPCTLService_InitLogger` and `DPCTLService_ShutdownLogger` functions during development to initialize the Google's logging library and de-initialize accordingly
+
```

Comment:
I do not like the suggested usage. The majority of dpctl's users are likely going to be Python native extension or library writers. The `DPCTLService_InitLogger` and `DPCTLService_ShutdownLogger` should have Python wrappers so that the logging behavior can be triggered via Python/Cython to aid in extension development.

Reply:
This is not addressed too. A new Cython extension added `dpctl._dev` that is not being loaded by `import dpctl`. It has 2 functions: `init_logger` and `fini_logger` and a context manager `verbose`.